frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -1,17 +1,26 @@
+import type { ReactNode } from "react";
 import { useCallback, useMemo } from "react";
 
 import {
   getColumnGroupIcon,
   getColumnGroupName,
 } from "metabase/common/utils/column-groups";
 import { getColumnIcon } from "metabase/common/utils/columns";
+import {
```

Comment:
nit (a11y): it's not possible to select and copy the text in the popover because it disappears when I press and hold left mouse button



https://github.com/metabase/metabase/assets/6830683/1106b24b-7f4d-4cc1-b580-e370e1fe499f